src/Utilities/MasterSlaveConfigParser.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace LaravelDoctrine\ORM\Utilities;
+
+use UnexpectedValueException;
+
+class MasterSlaveConfigParser
```

Comment:
@fureszpeter I don't think this should be extracted to a whole class.
This could live inside the `EntityManagerFactory`, as no other code will need to call it. Plus, a big, *big*, *BIG* :-1: to utility (static) classes.